repository-data/webfiles/src/main/resources/site/css/demo.css

```diff
@@ -96,3 +96,136 @@ input {
     transition: opacity .3s;
     width: 3em;
 }
+
+/* Pagination styling from Bootstrap css. To replace when styling is defined for site */
+.pagination {
+    display: inline-block;
+    padding-left: 0;
+    margin: 20px 0;
+    border-radius: 4px;
+}
+
+.pagination > li {
```

Comment:
no need for ">" here and anywhere in this CSS

Reply:
Will remove from css